### PR TITLE
Add android-studio-sync-base.conf

### PR DESCRIPTION
**Pull request by asodja**

### CODE REVIEW

File: android-studio-sync-base.conf [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# Performance scenario for testing Android Studio sync performance.
+#
+# Usage: (See also README.md)
+#
+# It requires Android Studio Bumblebee (2021.1.1) or greater
+#
+# 1. Create a project-specific scenario file in the parent directory.
+# 2. Include this file by adding the following in the project-specific scenario file:
+#
+#     include required(file("gradle-benchmark-base/android-studio-sync-base.conf"))
```
Comment (wolfs) [inline android-studio-sync-base.conf:10]: Can you include both incremental-base and android-studio-sync base?

**Review by wolfs — APPROVED** [review]
LGTM!

File: android-studio-sync-base.conf (hunk 1 above)
Comment (asodja) [reply android-studio-sync-base.conf:10]: I tested it and it works.

**Comment by asodja** [comment]
I will merge this when we merge also the last part, dependency mutator, to the gradle-profiler.

**Pull request merged**